docs/source/control_status_registers.rst

```diff
@@ -1714,10 +1716,10 @@ Reset Value: Defined
   :widths: 15 15 70
   :class: no-scrollbar-table
 
-  +-------------+-----------+----------------------------------------------------------------+
-  | **Bit #**   | **Mode**  | **Description**                                                |
-  +=============+===========+================================================================+
-  | 31:1        | RO        | Hardwired to 0.                                                |
-  +-------------+-----------+----------------------------------------------------------------+
-  | 0           | RO        | 1 if ``FPU`` = 1 and ``ZFINX`` = 1 else 0.                     |
-  +-------------+-----------+----------------------------------------------------------------+
+  +-------------+-----------+---------------------------------------------------+
+  | **Bit #**   | **Mode**  | **Description**                                   |
+  +=============+===========+===================================================+
+  | 31:1        | RO        | 0                                                 |
```

Comment:
why the mhartid depends on the FPU?

Reply:
This is not mhartid (line 1523) you see but Zfinx (line 1706) as you didn't expand 1522-1714 on the left side ...

Reply:
ops sorry :D